src/SprykerEco/Yves/Braintree/Theme/default/components/molecules/braintree-pay-pal/braintree-pay-pal.ts

```diff
@@ -7,83 +7,85 @@ export default class BraintreePayPal extends BraintreePaymentForm {
     protected submitBtn: HTMLElement;
 
     protected readonly formId: string = 'payment-form';
+    protected readonly paypalButtonSelector: string = '#paypal-button';
     protected readonly paymentMethodName: string = 'braintreePayPal';
     protected readonly paymentMethodTypeName: string = 'PayPalAccount';
     protected readonly nonceInputName: string = 'payment_method_nonce';
 
-    protected readyCallback(): void {
+    protected readyCallback(): void {}
+
+    protected init(): void {
         this.form = <HTMLFormElement>document.getElementById(`${this.formId}`);
         this.paymentMethods = <HTMLInputElement[]>Array.from(this.form.querySelectorAll(`input[name='${this.paymentSelection}']`));
         this.submitBtn = <HTMLElement>this.form.querySelector(`button[type='submit']`);
 
-        const tokenInput = document.createElement('input');
-        tokenInput.setAttribute('type', 'hidden');
-        tokenInput.setAttribute('name', `${this.nonceInputName}`);
-        this.form.appendChild(tokenInput);
-
-        var self = this;
-
+        super.createTokenField();
         this.mapEvents();
+        this.createBrainTreeClient();
+    }
 
-        // // Create a client.
+    protected createBrainTreeClient(): void {
         braintree.client.create({
             authorization: this.braintreeClientToken
-        }, function (clientErr, clientInstance) {
+        }, (clientError, clientInstance) => {
 
-            // Stop if there was a problem creating the client.
-            // This could happen if there is a network error or if the authorization
+            // stop if there was a problem creating the client.
```

Comment:
Suggest to remove comments. If you need to describe something - move it to a method or variable.